src/lib/modules/blockchain_listener/index.js

```diff
@@ -40,6 +43,50 @@ class BlockchainListener {
       this.processLogsApi.logHandler.handleLog({logLevel, message});
     });
   }
+
+  _registerConsoleCommands() {
+    this.embark.registerConsoleCommand({
+      description: 'Enables regular transactions to prevent transactions from getting stuck when using Geth and Metamask',
+      matches: ['regularTxs on'],
+      usage: "regularTxs on",
+      process: (cmd, callback) => {
+        this.events.request('regularTxs:start', callback);
+      }
+    });
+    this.embark.registerConsoleCommand({
+      description: 'Disables regular transactions to prevent transactions from getting stuck when using Geth and Metamask',
```

Comment:
```suggestion
      description: 'Disables regular transactions used to prevent transactions from getting stuck when using Geth and Metamask',
```
The description reads as if "Disabl[ing]" will "prevent transactions...".

Reply:
Great catch and makes sense - was a copy/paste issue :)